artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/jgroups/JChannelManager.java

```diff
@@ -34,7 +34,7 @@
 
    private static final Logger logger = Logger.getLogger(JChannelManager.class);
 
-   private Map<String, JChannelWrapper> channels;
```

Comment:
the factory itself was supposed to be singleton. I will make JChannelManager singleton on a separate commit. Let me handle it.